### PR TITLE
Fix/association post community

### PR DESCRIPTION
**Pull request by vasanthakumar-a**
Added association for new_community_post and changed the path

### CODE REVIEW

**Comment by vasanthakumar-a** [comment]
Resolved the conflicts and its up to date with master

File: app/views/posts/_list.html.erb [hunk 1]
```diff
@@ -40,27 +40,27 @@
...
+          <%= link_to community_post_path(post.community, post) do %>
           <div class="margin">
             <p class="col-12 sensitive-button">Click to see spoiler</p>
           </div>
           <% end %>
           <% elsif post.nsfw? %>
-          <%= link_to community_post_path(post.community_id, post) do %>
+          <%= link_to community_post_path(post.community, post) do %>
           <div class="margin">
             <p class="col-12 sensitive-button">Click to see nsfw</p>
```
Comment (iMagesh) [inline app/views/posts/_list.html.erb:65]: Can we move this to a helper?

**Comment by vasanthakumar-a** [comment]
Resolved the conflicts to master

File: app/views/posts/_list.html.erb (hunk 1 above)
Comment (vasanthakumar-a) [reply app/views/posts/_list.html.erb:65]: refactored this code

**Pull request merged**